sql/2024/seo/lighthouse-seo-stats-2024.sql

```diff
@@ -0,0 +1,59 @@
+CREATE TEMPORARY FUNCTION getAudits(audits STRING)
+RETURNS ARRAY<STRUCT<id STRING, weight INT64, title STRING, description STRING, score FLOAT64>> LANGUAGE js AS '''
+var auditsObj = JSON.parse(audits);
+var results = [];
+
+for (var auditId in auditsObj) {
+  if (auditsObj.hasOwnProperty(auditId)) {
+    var audit = auditsObj[auditId];
+    results.push({
+      id: auditId,
+      weight: audit.weight || 0,
+      title: audit.title || '',
+      description: audit.description || '',
+      score: audit.score || null
```

Comment:
This needs to be this:

```suggestion
      score: audit.score ?? null
```

Otherwise a score of `0` becomes `null` which then looks like the audit is not applicable.

Reply:
Reran the query and explained it to @fellowhuman1101

Think the last column is still a but suspect, the column names could do with a rename (done in sheet). and could do with a pct_using column (again done in sheet), but it's enough to unlock analysis for now.